gadget/gadget.go

```diff
@@ -264,18 +264,20 @@ type DiskStructureDeviceTraits struct {
 	OriginalKernelPath string `json:"kernel-path"`
 	// PartitionUUID is the partuuid as defined by i.e. /dev/disk/by-partuuid
 	PartitionUUID string `json:"partition-uuid"`
-	// FilesystemLabel is the label of the filesystem for structures that have
-	// filesystems, i.e. /dev/disk/by-label
-	FilesystemLabel string `json:"filesystem-label"`
 	// PartitionLabel is the label of the partition for GPT disks, i.e.
 	// /dev/disk/by-partlabel
 	PartitionLabel string `json:"partition-label"`
+	// PartitionType is the type of the partition i.e. 0x83 for ext4 on DOS, or
```

Comment:
0x83 does not necessarily imply ext4, but means a Linux filesystem in general, usually was ext2,3,4 but could also be other filesystems (reiserfs, xfs, and so on). AFAIU it's the same with GPT type.

Reply:
I tweaked this comment based on your feedback to unblock this. Will merge once it's green :)

Reply:
you beat me to it @mvo5 :smile: thanks for this change

Reply:
I really hope I did not duplicate work but took some work from your shoulders. If so happy to help if not my apologies.

Reply:
no worries just a single comment I hadn't committed yet